test/python/transpiler/test_dynamical_decoupling.py

```diff
@@ -339,6 +340,69 @@ def test_insert_dd_ghz_everywhere(self):
 
         self.assertEqual(ghz4_dd, expected)
 
+    def test_insert_dd_with_pulse_gate_calibrations(self):
```

Comment:
Do you also want a test for parametric echo gate? I don't think this is realistic and it's up to you.

Reply:
Since the code handles parameters, we might as well test them. I added an extra test.